configs/modules.config

```
@@ -230,4 +230,29 @@ process {
     //         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     //     ]
     // }
+    // MITOHIFI
+    // withName: 'FINDMITOREF' {
+    //     ext.args   = '--species${meta.species} --email${meta.email} --min_length${meta.min_length}'
```

Is this correct? Compare to other modules with parameters in `nextflow.config`.

---

Almost. If a variable in task context ( e.g. an input variable like `meta` ) needs to be accessed, then the string should be specified as a closure (anonymous function). If a closure is not used, Nextflow tries to evaluate the string when the config loads before workflow execution. A closure defers evaluation of the string until the task is actually executed.
```suggestion
    //     ext.args   = { '--species${meta.species} --email${meta.email} --min_length${meta.min_length}' }
```